include/functions.php

```diff
@@ -0,0 +1,220 @@
+				<a href='blogIndex.php'>Blog |</a>
+				<a href='projects.php'>Projects</a>
+			</div>
+	</body>
+";
+}
+
+
+function printNicely($interm) {
+	echo "<pre>";
+	var_dump($interm);
+	echo "<pre>";
+}
+
+
+//Blog functions!!
+
+function getAllBlogPosts() {
+	$result = dbQuery("
+		SELECT *
+		FROM blogPost
+	")->fetchAll();
+
+	return $result;
+}
+
+function getOneBlogPost($newBlogPostID) {
+	$result = dbQuery("
+		SELECT *
+		FROM blogPost
+		WHERE blogPostID = :blogPostID
+	", array(
+		'blogPostID'=>$newBlogPostID
+		))->fetch();
+
+	return $result;
+}
+
+
+//don't use this ever, probably. just use a foreach function
+function getNumPosts(){
+	$result = dbQuery("
+		SELECT COUNT(blogPostID) AS NumRows
+		FROM blogPost
+	")->fetch();
+	return $result;
+}
+
+//primary key will auto increment itself, don't need to add
+function insertBlogPost($title, $body, $dateCreated, $authorOfPost) {
+	$result = dbQuery("
+		INSERT INTO blogPost(title, body, dateCreated, authorOfPost)
+		VALUES('$title', '$body', '$dateCreated', '$authorOfPost')
+	")->fetch();
+}
+
+//deletes blog post by ID number, not any other factor
+function deleteBlogPost($newBlogPostID) {
+	$result = dbQuery("
+		DELETE FROM blogPost
+		WHERE blogPostID = :blogPostID
+		", array(
+			'blogPostID'=>$newBlogPostID
+		))->fetch();
+}
+
+
+
+//comment functions
+function getPostComments($thisBlogPostID){
+	$result = dbQuery("
+		SELECT *
+		FROM comments
+		WHERE blogPostID = :blogPostID
+		ORDER BY datePosted DESC
+		", array(
+			'blogPostID'=>$thisBlogPostID
+	))->fetchAll();
+	return $result;
+}
+
+
+function getAllTags(){
+	$result = dbQuery("
+		SELECT *
+		FROM tags
+		"
+	)->fetchAll();
+	return $result;
+}
+
+function getAllTagNames(){
+	$result = dbQuery("
+		SELECT DISTINCT tag
+		FROM tags
+		"
+	)->fetchAll();
+	return $result;
+}
+
+function getTagsWithThisName($oneTagName){
+	$result = dbQuery("
+		SELECT *
+		FROM tags
+		WHERE tag = :tag
+	", array(
+		'tag'=>$oneTagName
+	))->fetchAll();
+
+	return $result;
+}
+
+function getTagNamesWithThisNumber($oneTagNumber){
+	$result = dbQuery("
+		SELECT tag
+		FROM tags
+		WHERE blogPostID = :blogPostID
+	", array(
+		'blogPostID'=>$oneTagNumber
+	))->fetchAll();
+	return $result;
+}
+
+
+function getPostsWithThisAuthor($Author){
+	$result = dbQuery("
+		SELECT blogPostID
+		FROM blogPost
+		WHERE authorOfPost = :authorOfPost
+	", array(
+		'authorOfPost'=>$Author
+	))->fetchAll();
+	return $result;
+}
+
+function getAllAuthorNames(){
+	$result = dbQuery("
+		SELECT DISTINCT authorOfPost
+		FROM blogPost
+		"
+	)->fetchAll();
+	return $result;
+
+}
+
+
+
+function printBlogPost($var){
+
+echo "<body class='bg'>";
+	$post=getOneBlogPost($var);
+
+	echo "<div class='textStyle'>";
+		echo "<strong>Author: </strong>";
+		echo $post['authorOfPost'];
+		echo "<br>";
+		echo "<strong>Date Post Created: </strong>";
+		echo $post['dateCreated'];
+
+		$tagNames=getTagNamesWithThisNumber($post['blogPostID']);
```

Comment:
Back to the topic of consistency with how you name things, I think I prefer the `getPostComments` name below. If you followed that convention, this would be `getPostTags`